tests/test_bake_project.py

```diff
@@ -90,22 +90,22 @@ def test_bake_with_defaults(cookies):
 def test_bake_and_run_tests(cookies):
     with bake_in_temp_dir(cookies) as result:
         assert result.project.isdir()
-        run_inside_dir('python setup.py test', str(result.project)) == 0
+        run_inside_dir('python -m unittest discover', str(result.project)) == 0
         print("test_bake_and_run_tests path", str(result.project))
 
 
 def test_bake_withspecialchars_and_run_tests(cookies):
     """Ensure that a `full_name` with double quotes does not break setup.py"""
     with bake_in_temp_dir(cookies, extra_context={'full_name': 'name "quote" name'}) as result:
         assert result.project.isdir()
-        run_inside_dir('python setup.py test', str(result.project)) == 0
+        run_inside_dir('python setup.py --help', str(result.project)) == 0
```

Comment:
This doesn't do what the test name says

Reply:
You're right... When I saw the docstring I rationalized that the only reason it was running the tests was to validate that setup.py was formatted correctly. Do you agree/does this accomplish the same objective with less overhead? If so I'll rename it, if not I'll change it to run tests.